tests/slot-scenarios.html

```diff
@@ -188,7 +188,17 @@
             window.ShadyDOM.flush();
           }
         });
-      })
+      });
+
+      test('slots can be modified before rendering', function() {
```

Comment:
Let's go ahead and test if a node gets distributed to the right spot in this case.

Reply:
Done